elasticsearch/src/main/java/org/apache/calcite/adapter/elasticsearch/ElasticsearchTransport.java

```diff
@@ -279,22 +276,25 @@ private Response applyInternal(final HttpRequest request)
           request.getRequestLine().getMethod(),
           request.getRequestLine().getUri());
       r.setEntity(entity);
-      final Response response = restClient.performRequest(r);
 
-      final String payload = entity != null && entity.isRepeatable()
-          ? EntityUtils.toString(entity) : "<empty>";
+      try (RestClient restClient = clientBuilder.build()) {
```

Comment:
If I understand this correctly, for each unit test, we will create a new RestClient to request, which is what you mentioned.

- In my team, we manage the RestClients by Cache(for lifecycle and something else). Since it is actual business related, we might don't make it here
- For unit tests, I'm wandering if there is a suitable way to manage the RestClient in BeforeClass and AfterClass. Just my personal opinion.